src/tests/unit/tests/views/content/__snapshots__/content-panel.test.tsx.snap

```diff
@@ -1,23 +1,23 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ContentPanel renders closed 1`] = `
-"<StyledPanelBase isOpen={false} onDismiss={[undefined]} type={3} isLightDismiss={true} closeButtonAriaLabel=\\"Close panel\\">
+"<StyledPanelBase isOpen={false} onDismiss={[undefined]} type={3} isLightDismiss={true} closeButtonAriaLabel=\\"Close panel\\" headerText={[undefined]} headerTextProps={{...}} headerClassName=\\"content-header\\">
```

Comment:
It would be nice to update the test to pass a more representative value for the headerText. Maybe also nice would be to somehow write a test case verifying that all the infoAndExample content does actually specify a pageTitle, but not sure how challenging that would be to implement.

Reply:
Updated the test to pass in a value for pageTitle/headerText. I'm thinking about ways to verify the content--I'll open a new PR if I come up with anything